ibus.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=ibus daemon
+Documentation=man:ibus-daemon(1)
+
+[Service]
+Type=dbus
+BusName=org.freedesktop.IBus
+Environment=IBUS_SOCK=%t/ibus.socket
+ExecStart=@bindir@/ibus-daemon --replace --xim --panel disable -a unix:tmpdir=$IBUS_SOCK
```

Comment:
Please replace bus/org.freedesktop.IBus.service.in  with bus/bus.service.in.
Please replace '--xim' with @xim_cli_arg@
Please modify bus/Makefile.am with the install path.

I wonder if the ibus.service file can get an environment variable likes ibus-daemon --xim $IBUS_ARGS or can have a user directory likes $XDG_CONFIG_HOME/systemd/ibus.service